### DISCUSSION TITLE
Forge ModLauncher Fails to Start on Docker: Java Unsupported Version Error

### DISCUSSION THREAD

**Discussion by 1tpp** [itzg/docker-minecraft-server#3128] [General]
```yml
# docker-compose.yml
services:
  mc:
    image: itzg/minecraft-server
    ports:
      - "8221:25565"
    environment:
      EULA: "true"
      MODPACK_PLATFORM: AUTO_CURSEFORGE
      CF_API_KEY: ${CF_API_KEY}
      CF_SLUG: one-block-modded
      CF_MODPACK_ZIP: /modpacks/Modded+OneBlock.zip
      CF_SET_LEVEL_FROM: OVERRIDES
      DEBUG: "false"
      MEMORY: 8G
    volumes:
      - ./data:/data
      - ./modpacks:/modpacks:ro
    env_file: ".env"
```

```bash
 ✔ Container mc-oneblock-mc-1  Created                                                                                                                                                 0.0s
Attaching to mc-1
mc-1  | [init] Running as uid=1000 gid=1000 with /data as 'drwxr-xr-x 11 1000 1000 4096 Oct 28 00:08 /data'
mc-1  | [init] [WARN] Avoid setting TYPE and MODPACK_PLATFORM
mc-1  | [mc-image-helper] 00:13:22.842 INFO  : Requested CurseForge modpack Modded OneBock is already installed
mc-1  | [mc-image-helper] 00:13:23.617 INFO  : Forge version 43.1.38 for minecraft version 1.19.2 is already installed
mc-1  | [mc-image-helper] 00:13:25.602 INFO  : Created/updated 1 property in /data/server.properties
mc-1  | [init] Setting initial memory to 8G and max to 8G
mc-1  | [init] Using Forge supplied run.sh script...
mc-1  | 2024-10-28 00:13:27,807 main WARN Advanced terminal features are not available in this environment
mc-1  | [00:13:28] [main/INFO] [cp.mo.mo.Launcher/MODLAUNCHER]: ModLauncher running: args [--launchTarget, forgeserver, --fml.forgeVersion, 43.1.38, --fml.mcVersion, 1.19.2, --fml.forgeGroup, net.minecraftforge, --fml.mcpVersion, 20220805.130853]
mc-1  | [00:13:28] [main/INFO] [cp.mo.mo.Launcher/MODLAUNCHER]: ModLauncher 10.0.8+10.0.8+main.0ef7e830 starting: java version 21.0.5 by Eclipse Adoptium; OS Linux arch amd64 version 6.8.0-47-generic
mc-1  | [00:13:28] [main/INFO] [mixin/]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=union:/data/libraries/org/spongepowered/mixin/0.8.5/mixin-0.8.5.jar%2363!/ Service=ModLauncher Env=SERVER
mc-1  | [00:13:28] [main/WARN] [ne.mi.fm.lo.mo.ModFileParser/LOADING]: Mod file /data/libraries/net/minecraftforge/fmlcore/1.19.2-43.1.38/fmlcore-1.19.2-43.1.38.jar is missing mods.toml file
mc-1  | [00:13:28] [main/WARN] [ne.mi.fm.lo.mo.ModFileParser/LOADING]: Mod file /data/libraries/net/minecraftforge/javafmllanguage/1.19.2-43.1.38/javafmllanguage-1.19.2-43.1.38.jar is missing mods.toml file
mc-1  | [00:13:28] [main/WARN] [ne.mi.fm.lo.mo.ModFileParser/LOADING]: Mod file /data/libraries/net/minecraftforge/lowcodelanguage/1.19.2-43.1.38/lowcodelanguage-1.19.2-43.1.38.jar is missing mods.toml file
mc-1  | [00:13:28] [main/WARN] [ne.mi.fm.lo.mo.ModFileParser/LOADING]: Mod file /data/libraries/net/minecraftforge/mclanguage/1.19.2-43.1.38/mclanguage-1.19.2-43.1.38.jar is missing mods.toml file
mc-1  | [00:13:28] [main/INFO] [ne.mi.fm.lo.mo.JarInJarDependencyLocator/]: No dependencies to load found. Skipping!
mc-1  | [00:13:29] [main/INFO] [mixin/]: Compatibility level set to JAVA_17
mc-1  | [00:13:30] [main/INFO] [cp.mo.mo.LaunchServiceHandler/MODLAUNCHER]: Launching target 'forgeserver' with arguments []
mc-1  | [Serene Seasons Transformer]: Transforming m_8714_ (Lnet/minecraft/world/level/chunk/LevelChunk;I)V in net/minecraft/server/level/ServerLevel
mc-1  | [Serene Seasons Transformer]: Successfully patched tickChunk
mc-1  | [00:13:30] [main/WARN] [mixin/]: Error loading class: net/minecraft/server/level/ServerLevel (java.lang.IllegalArgumentException: Unsupported class file major version 65)
mc-1  | [00:13:30] [main/WARN] [mixin/]: @Mixin target net.minecraft.server.level.ServerLevel was not found chunkpregen.mixins.json:common.server.ServerLevelMixin
mc-1  | [00:13:30] [main/WARN] [mixin/]: Error loading class: vazkii/quark/addons/oddities/inventory/BackpackMenu (java.lang.ClassNotFoundException: vazkii.quark.addons.oddities.inventory.BackpackMenu)
mc-1  | [00:13:30] [main/WARN] [mixin/]: Error loading class: net/minecraft/world/item/ItemStack (java.lang.IllegalArgumentException: Unsupported class file major version 65)
mc-1  | [00:13:30] [main/WARN] [mixin/]: @Mixin target net.minecraft.world.item.ItemStack was not found placebo.mixins.json:ItemStackMixin
mc-1  | [00:13:30] [main/WARN] [mixin/]: Error loading class: java/lang/invoke/MethodHandles$Lookup (java.lang.IllegalArgumentException: Unsupported class file major version 65)
mc-1  | [00:13:30] [main/WARN] [mixin/]: Error loading class: java/lang/invoke/MethodHandles$Lookup (java.lang.IllegalArgumentException: Unsupported class file major version 65)
mc-1  | [00:13:30] [main/WARN] [mixin/]: Error loading class: java/util/Map$Entry (java.lang.IllegalArgumentException: Unsupported class file major version 65)
mc-1  | [00:13:30] [main/WARN] [mixin/]: Error loading class: java/lang/invoke/MethodHandles$Lookup (java.lang.IllegalArgumentException: Unsupported class file major version 65)
mc-1  | [00:13:30] [main/WARN] [mixin/]: Error loading class: java/lang/invoke/MethodHandles$Lookup (java.lang.IllegalArgumentException: Unsupported class file major version 65)
mc-1  | [00:13:30] [main/WARN] [mixin/]: Error loading class: java/lang/invoke/MethodHandles$Lookup (java.lang.IllegalArgumentException: Unsupported class file major version 65)
mc-1  | [00:13:30] [main/WARN] [mixin/]: Error loading class: java/util/concurrent/CompletableFuture (java.lang.IllegalArgumentException: Unsupported class file major version 65)
mc-1  | Exception in thread "main" java.lang.RuntimeException: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
mc-1  |         at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:32)
mc-1  |         at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53)
mc-1  |         at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71)
mc-1  |         at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.Launcher.run(Launcher.java:106)
mc-1  |         at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.Launcher.main(Launcher.java:77)
mc-1  |         at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
mc-1  |         at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
mc-1  |         at cpw.mods.bootstraplauncher@1.1.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)
mc-1  | Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
mc-1  |         at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:392)
mc-1  |         at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:250)
mc-1  |         at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.service.modlauncher.MixinTransformationHandler.processClass(MixinTransformationHandler.java:131)
mc-1  |         at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.launch.MixinLaunchPluginLegacy.processClass(MixinLaunchPluginLegacy.java:131)
mc-1  |         at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.serviceapi.ILaunchPluginService.processClassWithFlags(ILaunchPluginService.java:156)
mc-1  |         at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:88)
mc-1  |         at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120)
mc-1  |         at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.TransformingClassLoader.maybeTransformClassBytes(TransformingClassLoader.java:50)
mc-1  |         at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.readerToClass(ModuleClassLoader.java:113)
mc-1  |         at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.lambda$findClass$15(ModuleClassLoader.java:219)
mc-1  |         at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.loadFromModule(ModuleClassLoader.java:229)
mc-1  |         at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.findClass(ModuleClassLoader.java:219)
mc-1  |         at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:135)
mc-1  |         at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
mc-1  |         at java.base/java.lang.Class.getDeclaredMethods0(Native Method)
mc-1  |         at java.base/java.lang.Class.privateGetDeclaredMethods(Unknown Source)
mc-1  |         at java.base/java.lang.Class.getMethodsRecursive(Unknown Source)
mc-1  |         at java.base/java.lang.Class.getMethod0(Unknown Source)
mc-1  |         at java.base/java.lang.Class.getMethod(Unknown Source)
mc-1  |         at MC-BOOTSTRAP/fmlloader@1.19.2-43.1.38/net.minecraftforge.fml.loading.targets.CommonServerLaunchHandler.lambda$launchService$0(CommonServerLaunchHandler.java:29)
mc-1  |         at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30)
mc-1  |         ... 7 more
mc-1  | Caused by: org.spongepowered.asm.mixin.throwables.ClassMetadataNotFoundException: java.util.concurrent.CompletableFuture
mc-1  |         at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.transformMethod(MixinPreProcessorStandard.java:754)
mc-1  |         at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.transform(MixinPreProcessorStandard.java:739)
mc-1  |         at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attach(MixinPreProcessorStandard.java:310)
mc-1  |         at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.createContextFor(MixinPreProcessorStandard.java:280)
mc-1  |         at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinInfo.createContextFor(MixinInfo.java:1288)
mc-1  |         at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:292)
mc-1  |         at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:383)
mc-1  |         at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:365)
mc-1  |         at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363)
mc-1  |         ... 27 more
mc-1  | 2024-10-28T00:13:31.107Z        WARN    mc-server-runner        Minecraft server failed. Inspect logs above for errors that indicate cause. DO NOT report this line as an error.   {"exitCode": 1}
mc-1  | 2024-10-28T00:13:31.108Z        INFO    mc-server-runner        Done
mc-1 exited with code 1
```

**Comment by itzg** [comment]
Re-pull latest image to get proper Java version

https://docker-minecraft-server.readthedocs.io/en/latest/versions/java/#class-file-version-650

**Comment by minermartijn** [comment]
I got the same error, and repulling the latest image doenst work.  (Eddit, added my own error)

`OneBlockMineCraft  | [init] Running as uid=1000 gid=1000 with /data as 'drwxr-x--- 10 1000 1000 4096 Dec 16 12:20 /data'
OneBlockMineCraft  | [mc-image-helper] 12:26:30.315 INFO  : Requested CurseForge modpack Modded OneBock is already installed
OneBlockMineCraft  | [mc-image-helper] 12:26:30.726 INFO  : Forge version 43.1.38 for minecraft version 1.19.2 is already installed
OneBlockMineCraft  | [mc-image-helper] 12:26:33.148 INFO  : Created/updated 1 property in /data/server.properties
OneBlockMineCraft  | [init] Setting initial memory to 4G and max to 4G
OneBlockMineCraft  | [init] Using Forge supplied run.sh script...
OneBlockMineCraft  | 2024-12-16 12:26:36,268 main WARN Advanced terminal features are not available in this environment
OneBlockMineCraft  | [12:26:36] [main/INFO] [cp.mo.mo.Launcher/MODLAUNCHER]: ModLauncher running: args [--launchTarget, forgeserver, --fml.forgeVersion, 43.1.38, --fml.mcVersion, 1.19.2, --fml.forgeGroup, net.minecraftforge, --fml.mcpVersion, 20220805.130853]
OneBlockMineCraft  | [12:26:36] [main/INFO] [cp.mo.mo.Launcher/MODLAUNCHER]: ModLauncher 10.0.8+10.0.8+main.0ef7e830 starting: java version 21.0.5 by Eclipse Adoptium; OS Linux arch amd64 version 5.15.0-122-generic
OneBlockMineCraft  | [12:26:36] [main/INFO] [mixin/]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=union:/data/libraries/org/spongepowered/mixin/0.8.5/mixin-0.8.5.jar%2363!/ Service=ModLauncher Env=SERVER
OneBlockMineCraft  | [12:26:37] [main/WARN] [ne.mi.fm.lo.mo.ModFileParser/LOADING]: Mod file /data/libraries/net/minecraftforge/fmlcore/1.19.2-43.1.38/fmlcore-1.19.2-43.1.38.jar is missing mods.toml file
OneBlockMineCraft  | [12:26:37] [main/WARN] [ne.mi.fm.lo.mo.ModFileParser/LOADING]: Mod file /data/libraries/net/minecraftforge/javafmllanguage/1.19.2-43.1.38/javafmllanguage-1.19.2-43.1.38.jar is missing mods.toml file
OneBlockMineCraft  | [12:26:37] [main/WARN] [ne.mi.fm.lo.mo.ModFileParser/LOADING]: Mod file /data/libraries/net/minecraftforge/lowcodelanguage/1.19.2-43.1.38/lowcodelanguage-1.19.2-43.1.38.jar is missing mods.toml file
OneBlockMineCraft  | [12:26:37] [main/WARN] [ne.mi.fm.lo.mo.ModFileParser/LOADING]: Mod file /data/libraries/net/minecraftforge/mclanguage/1.19.2-43.1.38/mclanguage-1.19.2-43.1.38.jar is missing mods.toml file
OneBlockMineCraft  | [12:26:37] [main/INFO] [ne.mi.fm.lo.mo.JarInJarDependencyLocator/]: No dependencies to load found. Skipping!
OneBlockMineCraft  | [12:26:40] [main/INFO] [mixin/]: Compatibility level set to JAVA_17
OneBlockMineCraft  | [12:26:40] [main/INFO] [cp.mo.mo.LaunchServiceHandler/MODLAUNCHER]: Launching target 'forgeserver' with arguments []
OneBlockMineCraft  | [Serene Seasons Transformer]: Transforming m_8714_ (Lnet/minecraft/world/level/chunk/LevelChunk;I)V in net/minecraft/server/level/ServerLevel
OneBlockMineCraft  | [Serene Seasons Transformer]: Successfully patched tickChunk
OneBlockMineCraft  | [12:26:40] [main/WARN] [mixin/]: Error loading class: net/minecraft/server/level/ServerLevel (java.lang.IllegalArgumentException: Unsupported class file major version 65)
OneBlockMineCraft  | [12:26:40] [main/WARN] [mixin/]: @Mixin target net.minecraft.server.level.ServerLevel was not found chunkpregen.mixins.json:common.server.ServerLevelMixin
OneBlockMineCraft  | [12:26:40] [main/WARN] [mixin/]: Error loading class: vazkii/quark/addons/oddities/inventory/BackpackMenu (java.lang.ClassNotFoundException: vazkii.quark.addons.oddities.inventory.BackpackMenu)
OneBlockMineCraft  | [12:26:40] [main/WARN] [mixin/]: Error loading class: net/minecraft/world/item/ItemStack (java.lang.IllegalArgumentException: Unsupported class file major version 65)
OneBlockMineCraft  | [12:26:40] [main/WARN] [mixin/]: @Mixin target net.minecraft.world.item.ItemStack was not found placebo.mixins.json:ItemStackMixin
OneBlockMineCraft  | [12:26:40] [main/WARN] [mixin/]: Error loading class: java/lang/invoke/MethodHandles$Lookup (java.lang.IllegalArgumentException: Unsupported class file major version 65)
OneBlockMineCraft  | [12:26:40] [main/WARN] [mixin/]: Error loading class: java/lang/invoke/MethodHandles$Lookup (java.lang.IllegalArgumentException: Unsupported class file major version 65)
OneBlockMineCraft  | [12:26:40] [main/WARN] [mixin/]: Error loading class: java/util/Map$Entry (java.lang.IllegalArgumentException: Unsupported class file major version 65)
OneBlockMineCraft  | [12:26:40] [main/WARN] [mixin/]: Error loading class: java/lang/invoke/MethodHandles$Lookup (java.lang.IllegalArgumentException: Unsupported class file major version 65)
OneBlockMineCraft  | [12:26:40] [main/WARN] [mixin/]: Error loading class: java/lang/invoke/MethodHandles$Lookup (java.lang.IllegalArgumentException: Unsupported class file major version 65)
OneBlockMineCraft  | [12:26:40] [main/WARN] [mixin/]: Error loading class: java/lang/invoke/MethodHandles$Lookup (java.lang.IllegalArgumentException: Unsupported class file major version 65)
OneBlockMineCraft  | [12:26:41] [main/WARN] [mixin/]: Error loading class: java/util/concurrent/CompletableFuture (java.lang.IllegalArgumentException: Unsupported class file major version 65)
OneBlockMineCraft  | Exception in thread "main" java.lang.RuntimeException: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
OneBlockMineCraft  | 	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:32)
OneBlockMineCraft  | 	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53)
OneBlockMineCraft  | 	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71)
OneBlockMineCraft  | 	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.Launcher.run(Launcher.java:106)
OneBlockMineCraft  | 	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.Launcher.main(Launcher.java:77)
OneBlockMineCraft  | 	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
OneBlockMineCraft  | 	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
OneBlockMineCraft  | 	at cpw.mods.bootstraplauncher@1.1.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)
OneBlockMineCraft  | Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
OneBlockMineCraft  | 	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:392)
OneBlockMineCraft  | 	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:250)
OneBlockMineCraft  | 	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.service.modlauncher.MixinTransformationHandler.processClass(MixinTransformationHandler.java:131)
OneBlockMineCraft  | 	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.launch.MixinLaunchPluginLegacy.processClass(MixinLaunchPluginLegacy.java:131)
OneBlockMineCraft  | 	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.serviceapi.ILaunchPluginService.processClassWithFlags(ILaunchPluginService.java:156)
OneBlockMineCraft  | 	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:88)
OneBlockMineCraft  | 	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120)
OneBlockMineCraft  | 	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.TransformingClassLoader.maybeTransformClassBytes(TransformingClassLoader.java:50)
OneBlockMineCraft  | 	at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.readerToClass(ModuleClassLoader.java:113)
OneBlockMineCraft  | 	at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.lambda$findClass$15(ModuleClassLoader.java:219)
OneBlockMineCraft  | 	at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.loadFromModule(ModuleClassLoader.java:229)
OneBlockMineCraft  | 	at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.findClass(ModuleClassLoader.java:219)
OneBlockMineCraft  | 	at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:135)
OneBlockMineCraft  | 	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
OneBlockMineCraft  | 	at java.base/java.lang.Class.getDeclaredMethods0(Native Method)
OneBlockMineCraft  | 	at java.base/java.lang.Class.privateGetDeclaredMethods(Unknown Source)
OneBlockMineCraft  | 	at java.base/java.lang.Class.getMethodsRecursive(Unknown Source)
OneBlockMineCraft  | 	at java.base/java.lang.Class.getMethod0(Unknown Source)
OneBlockMineCraft  | 	at java.base/java.lang.Class.getMethod(Unknown Source)
OneBlockMineCraft  | 	at MC-BOOTSTRAP/fmlloader@1.19.2-43.1.38/net.minecraftforge.fml.loading.targets.CommonServerLaunchHandler.lambda$launchService$0(CommonServerLaunchHandler.java:29)
OneBlockMineCraft  | 	at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.8/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30)
OneBlockMineCraft  | 	... 7 more
OneBlockMineCraft  | Caused by: org.spongepowered.asm.mixin.throwables.ClassMetadataNotFoundException: java.util.concurrent.CompletableFuture
OneBlockMineCraft  | 	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.transformMethod(MixinPreProcessorStandard.java:754)
OneBlockMineCraft  | 	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.transform(MixinPreProcessorStandard.java:739)
OneBlockMineCraft  | 	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attach(MixinPreProcessorStandard.java:310)
OneBlockMineCraft  | 	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.createContextFor(MixinPreProcessorStandard.java:280)
OneBlockMineCraft  | 	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinInfo.createContextFor(MixinInfo.java:1288)
OneBlockMineCraft  | 	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:292)
OneBlockMineCraft  | 	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:383)
OneBlockMineCraft  | 	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:365)
OneBlockMineCraft  | 	at MC-BOOTSTRAP/org.spongepowered.mixin/org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363)`
